### PR TITLE
Sparkmax controller group

### PR DESCRIPTION
**Pull request by AndrewItIs**
This branch is a robot base code with a Sparkmax+neo drive system.

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -6,14 +6,7 @@
 			<attribute name="gradle_used_by_scope" value="main,test"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="src" output="bin/test" path="src/test/java">
-		<attributes>
-			<attribute name="gradle_scope" value="test"/>
-			<attribute name="gradle_used_by_scope" value="test"/>
```
Comment (bryanduerk) [inline .classpath]: Why did we delete this

File: build.gradle [hunk 2]
```diff
@@ -1,102 +1,111 @@
-plugins {
-    id "java"
-    id "edu.wpi.first.GradleRIO" version "2020.1.2"
-}
-
-sourceCompatibility = JavaVersion.VERSION_13
-targetCompatibility = JavaVersion.VERSION_13
-
-def ROBOT_MAIN_CLASS = "frc.robot.Main"
-
```
Comment (bryanduerk) [inline build.gradle]: Revert this file

File: settings.gradle [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
-        if (OperatingSystem.current().isWindows()) {
-            String publicFolder = System.getenv('PUBLIC')
-            if (publicFolder == null) {
-                publicFolder = "C:\\Users\\Public"
-            }
-            def homeRoot = new File(publicFolder, "wpilib")
-            frcHome = new File(homeRoot, frcYear)
-        } else {
-            def userFolder = System.getProperty("user.home")
-            def homeRoot = new File(userFolder, "wpilib")
```
Comment (bryanduerk) [inline settings.gradle]: revert

File: src/main/java/frc/robot/RobotMap.java [hunk 4]
```diff
@@ -111,7 +111,8 @@
   public static final boolean useRemoteImu = false;
 
   public enum RobotId {
-    MINIBOT, ROBOT_2018, ROBOT_2019
+    MINIBOT, COMPETITION_2_2020, COMPETITION_1_2020
```
Comment (bryanduerk) [inline src/main/java/frc/robot/RobotMap.java]: Lets go with PracticeBot and CompetetionBot

Comment (arfrie22) [reply src/main/java/frc/robot/RobotMap.java]: I updated them to KITBOT and ROBOT_2020

File: settings.gradle (hunk 3 above)
Comment (arfrie22) [reply settings.gradle]: It got moved in the file

File: build.gradle (hunk 2 above)
Comment (arfrie22) [reply build.gradle]: Got moved, and we need to go with java 11 until they update the rio

File: .classpath (hunk 1 above)
Comment (arfrie22) [reply .classpath]: Not a human-controlled file, was auto done by either vscode or gradle.

**Pull request merged**